src/engine/Engine.h

```diff
@@ -126,6 +126,11 @@ class Engine : public IEngine, public SignalHandler::Signalable
     */
     void applySplit( const PiecewiseLinearCaseSplit &split );
 
+    /*
+      Recompute status of basic variables
+    */
+    void recomputeBasicStatus() { _tableau->computeBasicStatus(); };
```

Comment:
This doesn't seem like a method that should appear in the Engine.
Perhaps we should only expose a Tableau method called "contextPopHook", or something like that, that will get invoked after a pop?

Reply:
Great point, done!